crates/core/src/dht/dht_store.rs

```diff
@@ -276,16 +260,10 @@ impl DhtStore {
         self.meta_storage.write().unwrap().add_eavi(&eavi)
     }
 
-    pub(crate) fn get_entry_from_cas(
-        &self,
-        address: &Address,
-    ) -> Result<Option<Entry>, HolochainError> {
-        if let Some(json) = self.cas_fetch(&address)? {
-            let entry = Entry::try_from_content(&json)?;
-            Ok(Some(entry))
-        } else {
-            Ok(None) // no errors but entry is not in CAS
-        }
+    // Getters (for reducers)
+    // =======
+    pub(crate) fn meta_storage(&self) -> Arc<RwLock<dyn EntityAttributeValueStorage<Attribute>>> {
```

Comment:
One of the main fixes of https://github.com/holochain/holochain-rust/pull/1852 was to remove this function (and `content_storage()`) to encapsulate the lock inside the state and not pass around references to the lock or it's lock guards. This reverts that.

Reply:
Agree that this should be removed.

Reply:
Turns out this was only used once internally so it was easy to drop it completely